tests/cpp/relay/backend/graph_plan_token_alloc.cc

```diff
@@ -24,23 +24,24 @@
 namespace tvm {
 namespace relay {
 
-// TokenAllocator2d is necessary because in class TokenAllocator2D we don't
+// TokenAllocatorMixed is necessary because in class TokenAllocatorMixed we don't
 // have an access to its protected members. In this class we add new methods
-// which allow us to get and check internal state of class TokenAllocator2D
-class TokenAllocator2DWrapper : public TokenAllocator2D {
+// which allow us to get and check internal state of class TokenAllocatorMixed
+class TokenAllocatorMixedWrapper : public TokenAllocatorMixed {
  public:
-  inline size_t FreeListSize() const { return free_list_.size(); }
-  inline size_t BlockMapSize() const { return blocks_.size(); }
+  inline size_t FreeListSize() const { return free_.size(); }
+  inline size_t AllocListSize() const { return data_.size(); }
 };
 
-TEST(Token2DAlloc, OneToken) {
-  TokenAllocator2DWrapper alloc;
+TEST(TokenMixedAlloc, OneToken) {
+  TokenAllocatorMixedWrapper alloc;
   int storage_ids = 0;
-  EXPECT_EQ(alloc.BlockMapSize(), 0);
+  EXPECT_EQ(alloc.AllocListSize(), 0);
   EXPECT_EQ(alloc.FreeListSize(), 0);
 
   TensorType tt1({1, 22, 20, 20, 4}, DataType(kDLFloat, 32, 1));
-  VirtualDevice vd1(kDLOpenCL, 0, {}, MemoryScope("global.texture-nhwc"));
+  VirtualDevice vd1(kDLOpenCL, 0, Target("opencl -device=adreno"),
```

Comment:
I'd like to see the same number of tests for `opencl` target (w/o adreno). In this case we can check that token allocator works for all scenarios. You can do that by using [parameterized tests](http://google.github.io/googletest/advanced.html#how-to-write-value-parameterized-tests).